lib/forklift/box_factory.rb

```diff
@@ -30,6 +32,11 @@ def add_boxes(box_file, version_file)
 
     private
 
+    def load_box_file(file)
+      file = File.read(file)
+      YAML.safe_load(ERB.new(file).result, [Regexp], [], true)
```

Comment:
I wanted to be able to use environment variables for defining paths for mounts, as the example shows, passing it through ERB is the simples way to get that.